app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,40 @@
+class GroupsController < ApplicationController
+  before_action :authenticate_user!, only: %i[new create]
+
+  def index
+    @groups = current_user.groups
```

Comment:
- Kindly as the previous reviewer requested, I got an error saying NoMethodError in GroupsController#index. I would kindly advise you to check where the error is coming from and fix it to have a perfect working project.

Reply:
well resolved! ✔️